Python/Multiplicative Inverse.py

```diff
@@ -0,0 +1,44 @@
+def extended_euclidean(p, q):
+    """(int, int) -> list

```

Comment:
Doc Strings which start in the next line and not in running order are more readable.
```python
def function():
    '''
    explanation starts here
    '''
```